include/znc/Utils.h

```diff
@@ -152,9 +152,21 @@ class CException {
 +-------+-------+
 | hello | world |
 +-------+-------+@endverbatim
+ *
+ *  ChanServ-style output can be generated like so (only supports two columns!):
```

Comment:
I question the usefulness of this comment. ChanServ-style depends on services package and is subject to change in the future.

Reply:
Same a couple of lines up.

Reply:
I was copying the wording from the original issue (#914). To be frank, I was at loss for a good description, so I switched between "definition list" and "ChanServ style".

Reply:
Plain-style? Mobile-friendly?

Reply:
You could do a show-dont-tell approach and just paste a small sample of the output it would generate.

Reply:
`GridStyle` v `ListStyle`, maybe? (Note that this is not a full replacement: you can only use it on two-column-tables. Things like `/znc ListNetworks` will for now stay at the old style)

Reply:
> You could do a show-dont-tell approach 

the GridStyle table also has such a sentence in front of it ("The above code would generate the following output:") and it helps to clarify the difference between the two output formats. 

I don't really see the point in arguing about such details right now if I'm honest. (If "ListStyle" is OK, I'll update all documentation to use the new term, of course)

Reply:
Oh yeah, it's just a very minor point. It's part of the reason I deleted the comment :) - More of a "if we can quickly think of something better, go wild, if not - eh, alright"